### PR TITLE
Make eslint work universally both for JS and TS code

### PR DESCRIPTION
**Pull request by nazar-pc**
I got tired with IDE not working with eslint config that requires environment variables, so I made it "just work" universally without any.

### CODE REVIEW

File: node/.eslintrc.js [hunk 1]
```diff
@@ -175,71 +175,59 @@ const eslintConfig =
...
+});
 
-	default:
-	{
-		throw new TypeError('wrong/missing MEDIASOUP_NODE_LANGUAGE env');
-	}
-}
+eslintConfig.overrides.push({
+	files : [ '*.js' ],
+	env   : {
```
Comment (ibc) [inline node/.eslintrc.js]: The initial bracket should be in next line according to our ESlint rules so something is not working here if this "error" is not detected and CI passes, do I miss something?

Comment (nazar-pc) [reply node/.eslintrc.js]: I do not think it was enforced by eslint before, you might want to add/change rule for that

Comment (ibc) [reply node/.eslintrc.js]: Just please edit any .ts file and add a { in the same line as the = and you'll see 😉

Comment (nazar-pc) [reply node/.eslintrc.js]: I tried, it didn't complain either (tried before and after this PR).

Comment (ibc) [reply node/.eslintrc.js]: Perhaps the rule just applies to TS code

**Review by ibc — APPROVED** [review]
We should move Node tests to TS so this PR goes in the good direction. Thanks.

**Pull request merged**